download.sh

```
@@ -0,0 +1,14 @@
+CONTAINER_NAME="pet-skin-disease-data"  # 다운로드할 컨테이너 이름
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)